task_processing/plugins/mesos/logging_executor.py

```diff
@@ -171,6 +171,10 @@ def event_loop(self):
                     self.staging_tasks = self.staging_tasks.set(e.task_id, url)
 
                 if e.kind == 'task' and e.platform_type == 'running':
+                    if e.task_id not in self.staging_tasks:
+                        log.warning(f"Not tracking logs for task {e.task_id}")
```

Comment:
i would make this a debug statement "Task already running" or something, recovery and reconciliation should be "normal" events, not something that triggers warnings